Cargo.toml

```diff
@@ -32,3 +32,7 @@ halo2_proofs = { git = "https://github.com/powdr-org/halo2", branch = "kilic/shu
 # TODO change back to this once the PR is merged
 #halo2_proofs = { git = "https://github.com/appliedzkp/halo2.git", rev = "d3746109d7d38be53afc8ddae8fdfaf1f02ad1d7" }
 halo2_proofs = { git = "https://github.com/powdr-org/halo2", branch = "kilic/shuffle" }
+
+[profile.release-with-debug]
```

Comment:
Is this needed or just for debugging in this PR?

Reply:
This was used just to debug this PR, but it is useful to debug anything at full speed, and you only use it if you want, so I saw no harm in leaving it.